Xlib/protocol/display.py

```diff
@@ -25,6 +25,9 @@
 import errno
 import socket
 
+# Python 2/3 compatibility.
+from six import PY3, byte2int, indexbytes
```

Comment:
I can change those, but this is something that is explicitly mentioned as allowed in PEP8:

> It's okay to say this though:
> 
> `from subprocess import Popen, PIPE`